backend/src/api/routes/users/index.ts

```diff
@@ -0,0 +1,34 @@
+import express from "express";
+import {Theme, FetchedUser} from "codetierlist-types";
+import prisma from "../../../common/prisma";
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+    res.send(req.user satisfies FetchedUser);
+});
+
+/**
+ * Sets the user's theme to the specified theme.
+ * @param theme the theme to set. Must be "light" or "dark".
+ * @returns 200 if success, 400 if fail
+ */
+router.post("/set-theme", async (req, res) => {
+    if (!req.body.theme) {
+        res.status(400).send("No theme specified.");

```

Comment:
```suggestion
        res.status(400).send({message:"No theme specified."});

```
Use json to be consistent with the rest of the api